### PR TITLE
Ensure linter checks for generated code too

### PR DESCRIPTION
**Pull request by ccremer**
## Summary

* `lint` when executed from GH action also checks if generated code is up-to-date.
* This prevents doing changes but forgetting to run `make generate`

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`,
      as they show up in the changelog

### CODE REVIEW

File: config/crd/apiextensions.k8s.io/v1/base/backup.appuio.ch_effectiveschedules.yaml [hunk 1]
```diff
@@ -18,11 +18,11 @@ spec:
   versions:
   - additionalPrinterColumns:
     - description: Schedule Namespace
-      jsonPath: .spec.effectiveSchedules[0].namespace
+      jsonPath: .spec.scheduleRefs[0].namespace
```
Comment (Kidswiss) [inline config/crd/apiextensions.k8s.io/v1/base/backup.appuio.ch_effectiveschedules.yaml:21]: Can you put the generated things in their own commit?

**Review by Kidswiss — CHANGES_REQUESTED** [review]

File: config/crd/apiextensions.k8s.io/v1/base/backup.appuio.ch_effectiveschedules.yaml (hunk 1 above)
Comment (ccremer) [reply config/crd/apiextensions.k8s.io/v1/base/backup.appuio.ch_effectiveschedules.yaml:21]: sure thing

**Review by tobru — APPROVED** [review]
